web/index.html

```diff
@@ -63,5 +60,15 @@ <h1>Title of the chart</h1>
       </div>
     </div>
   </footer>
+
+  <!-- Login modal -->
+  <div class="modal-wrapper" id="login-modal">
+    <div class="modal-content">
+      <h5>This demo is meant to be private to Africa's voices.</h5>
```

Comment:
Let's make this a bit more intentional.

"This demo is only accessible to Africa's Voices. Please reach out if you'd like to see a demo"

Reply:
Sure, noted

Reply:
Fixed